### PR TITLE
Update client tests to default to pointing to v1.0.155

### PR DESCRIPTION
**Pull request by snf2ye**

### CODE REVIEW

**Review by nmalfroy — APPROVED** [review]

**Review by marikomedlock — COMMENTED** [review]
Thank you!

**Review by marikomedlock — APPROVED** [review]

**Review by nmalfroy — APPROVED** [review]

File: datarepo-clienttests/src/main/resources/servers/shdev.json [hunk 1]
```diff
@@ -15,6 +15,6 @@
     "parameters": ["https://raw.githubusercontent.com/broadinstitute/datarepo-helm-definitions/master/dev/sh/datarepo-api.yaml"]
   },
   "testRunnerServiceAccountFile": "jade-k8-sa.json",
-  "skipDeployment": false,
+  "skipDeployment": true,
```
Comment (fboulnois) [inline datarepo-clienttests/src/main/resources/servers/shdev.json:18]: Could you comment why this change is necessary? Thanks!

**Review by fboulnois — APPROVED** [review]
lgtm, one question:

File: datarepo-clienttests/src/main/resources/servers/shdev.json (hunk 1 above)
Comment (snf2ye) [reply datarepo-clienttests/src/main/resources/servers/shdev.json:18]: It's not - it's just a change on for when I'm running against my personal dev env. I decided I'd rather the default be to skip deployment.
